acre.yaml

```diff
@@ -704,6 +704,10 @@ validation:
             - - optionalfield
               - privateKey
               - privatekey
+          - - optionalfield
```

Comment:
Was this line added by accident? If not, what is it supposed to do?

Reply:
By accident, thanks for catching it!